_docs/setup/kubernetes/ansible-install.md

```diff
@@ -0,0 +1,121 @@
+
+Remark : Furthermore the minimum Kubernetes version that is compatible is `1.7.0` (`3.7.0` is the corresponding OpenShift version).   
+
+**Important**: All invocations of the Ansible playbooks need to take place at the `install/ansible` path of the project.
+Failing to do so will result in unexpected errors 
+
+The simplest execution command looks like the following:
+ 
+```bash
+ansible-playbook main.yml
+```
+
+Remarks:
+- The role tries it's best to be idempotent, so running the playbook multiple times should be have the same effect as running it a single time.   
+- The default parameters that apply to this role can be found in `istio/defaults/main.yml`.
+
+The full list of configurable parameters is as follows:
+
+| Parameter | Description | Values |
+| --- | --- | --- |
+| `cluster_flavour` | Defines whether the target cluster is a Kubernetes or an Openshift cluster. | Valid values are `k8s` and `ocp` (default) |
+| `github_api_token` | The API token used for authentication when calling the GitHub API | Any valid GitHub API token or empty (default) |
+| `cmd_path` | Can be used when the user does not have the `oc` or `kubectl` binary on the PATH | Defaults to expecting the binary is on the path | 
+| `istio.release_tag_name` | Should be a valid Istio release version. If left empty, the latest Istio release will be installed | `0.2.12`, `0.3.0`, `0.4.0` (default) |
+| `istio.dest` | Destination folder you want to install on your machine istio distribution | `~/.istio` (default) |
+| `istio.auth` | Boolean value to install istio using MUTUAL_TLS | `true` and `false` (default) |
+| `istio.namespace` | The namespace where istio will be installed | `istio-system` (default) |
+| `istio.addon` | Which Istio addons should be installed as well | This field is an array field, which by default contains `grafana`, `prometheus`, `zipkin` and `servicegraph` |
+| `istio.jaeger` | Whether or not Jaeger tracing should also be installed | `true` and `false` (default)|
+| `istio.delete_resources` | Boolean value to delete resources created under the istio namespace | `true` and `false` (default)|
+| `istio.samples` | Array containing the names of the samples that should be installed | Valid names are: `bookinfo`, `helloworld`, `httpbin`, `sleep`
+
+
+An example of an invocation where we want to deploy Jaeger instead of Zipkin would be:
+```bash
+ansible-playbook main.yml -e '{"istio": {"jaeger": true}}'
+```
+
+
+This playbook will take care of downloading and installing Istio locally on your machine, before deploying the necessary Kubernetes / Openshift
+pods, services etc. on to the cluster
+
+### Note on istio.delete_resources
+
+Activating the `istio.delete_resources` flag will result in any Istio related resources being deleted from the cluster before Istio is reinstalled.
+
+In order to avoid any inconsistency issues, this flag should only be used to reinstall the same version of Istio on a cluster. If a new version
```

Comment:
This whole section is confusing.  In my understanding, deleting the istio-system namespace will delete all objects within it, effectively uninstalling the Istio control plane.

Reply:
Of course deleting the `istio-system` namespace is the most effective way of uninstalling the control plane. 
The flag is used as a less "nuclear" option, since deleting / recreating the namespace can be (?) resource intensive